web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -59,20 +58,24 @@
     private boolean agentIsUp = false;
     private final LinkedList<ThreadLocalCleaner> cleaners = new LinkedList<ThreadLocalCleaner>();
 
+    //Only for Spring Boot Application (External Name Setting)
+    private String applicationName;
```

Comment:
@dhaval24 , please check where this name is used through the code and whether it makes it into the telemetry item at all. If not, keeping it as `applicationName` is ok.